lib/suse/connect/client.rb

```diff
@@ -94,6 +101,7 @@ def announce_system(distro_target = nil, instance_data_file = nil)
       # Re-send the system's hardware details on SCC
       #
       def update_system(distro_target = nil, instance_data_file = nil)
+        log.info "Update system details on SCC/Registration proxy ...\n".bold
```

Comment:
Suggest making the voice of this line match the rest of the output - "Updating"/"Updated" as you prefer